### ISSUE TITLE
[Buildkite Test Analytics] - Include Jetpack UI Tests in Buildkite Test Analytics

### ISSUE THREAD

**Issue by jostnes** [wordpress-mobile/WordPress-iOS#20336]
Currently Jetpack UI tests are not configured to appear on Buildkite Test Analytics. This issue is to track the work to add Test Analytics to Jetpack UI tests (both iPhone and iPad).

**Comment by mokagio** [comment]
I created the test suites in Buildkite and updated the tokens that the app use to point to those suites instead of the WordPress UI tests ones.

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/1218433/225510142-7aad1c70-7bcd-4143-83c0-b90766aed12a.png">

As long as we'll run only Jetpack, we should be fine.

We'll be able to close this issue once:

- [x] The workaround to skip Test Analytics in https://github.com/wordpress-mobile/WordPress-iOS/pull/20301 is removed
- [x] We have verified that data arrives to the new Jetpack Test Analytics suites.
